src/client/redirect-watch.js

```diff
@@ -8,69 +8,84 @@ import domProcessor from './dom-processor/index';
 const HASH_RE = /#.*$/;
 
 export default class RedirectWatch extends EventEmiter {
-    constructor (codeInstrumentation, eventSandbox) {
+    constructor (listeners, codeInstrumentation, elementSandbox) {
         super();
 
         this.DETECTED_EVENT = 'hammerhead|event|redirect-detected';
 
```

Comment:
Guys, the class name is already present `redirect`, it will be duplicated in the call:

`redirectWatch.DETECTED_EVENT`

or

`redirectWatch.REDIRECT_DETECTED_EVENT`